### ISSUE TITLE
T-Watch S3

### ISSUE THREAD

**Issue by danielktdoranie** [sharandac/My-TTGO-Watch#409]
I am new to ESP32 and LilyGO... I bought_ the T-Watch S3 I see this firmware doesn't support the T-Watch S3... could the T-Watch S3 be added? I updated my T-Watch S3 with the latest firmware and it looks the same... same ugly watch faces, I can see WiFi networks but not connect too them... just disappointing. If not sharandac is there any other firmware that supported the T-Watch S3? Thanks.

**Comment by donniebrasc** [comment]
As do i, s3 with lora. it would be great to see a mesh implement as well but ill take usefulness

**Comment by miguepollo** [comment]
I have a public fork of this repository and I am working on implement it
https://github.com/miguepollo/My-TTGO-Watch

**Comment by danielktdoranie** [comment]
> I have a public fork of this repository and I am working on implement it https://github.com/miguepollo/My-TTGO-Watch

That’s awesome news! Is it ready for public testing yet

**Comment by miguepollo** [comment]
No, because the clock restarts. But I'm working on getting it to work properly.

**Comment by miguepollo** [comment]
If you don't see much change, I am making changes to ttgo-t-watch-libraryS3 to make it work better.

**Comment by danielktdoranie** [comment]
> If you don't see much change, I am making changes to ttgo-t-watch-libraryS3 to make it work better.

Thanks so much for doing this!

**Comment by crt0mega** [comment]
> If you don't see much change, I am making changes to ttgo-t-watch-libraryS3 to make it work better.

Thank you! 🥳

**Comment by danielktdoranie** [comment]
> No because the clock restarts all the time. But I am working to make it work well.

Please could you let me know (maybe make a post here?) when you’re ready for beta testers? I will happily flash my T-Watch-S3 with beta firmware to help

**Comment by miguepollo** [comment]
Ok I'll let you know when it's ready for the beta-testers to try it :)

**Comment by ivanperez-keera** [comment]
Any updates on this?

Tagging @miguepollo .

**Comment by my-project-droid** [comment]
Can't you  just flash Meshtastic and Bruce firmware onto your lilygo T-watch S3

**Comment by ivanperez-keera** [comment]
@my-project-droid can you explain in more detail? Links?
